docs/join-the-network/nodes/validator.md

```diff
@@ -136,7 +137,7 @@ It is now time to set up your validator by doing the following:
    subject to slashing.
  - Select the Controller. This is the account that will decide when to start or stop validating.
 
-First, go to the **Developer** tab in the [Avail Apps](https://devnet-avail.polygon.technology/)
+First, go to the **Developer** tab in the [Avail Apps](https://devnet.avail.tools/)
```

Comment:
We do not have a devnet now. Do we want to point to testnet or devnet?

Reply:
I changed all 3 links in this file to the testnet. But as with the other pages, this one was just copied over, I didn't rewrite it (and it probably needs it).